---
name: Bug Report
about: Report a bug in the Multiwoven UI
title: "[BUG]: Brief Description" # e.g [BUG]: BigQuery Connector Authentication Failure
labels: 'bug'
assignees: ''
---

## Description
<!-- A clear and concise description of what the bug is e.g BigQuery connector fails to authenticate with valid credentials. Attach screenshots as well for more clarity. -->

## Steps to Reproduce
<!-- Steps to reproduce the behavior e.g. 1. Enter BigQuery credentials. 2. Attempt to connect. 3. Observe authentication failure. -->

## Expected Behavior
<!-- A clear and concise description of what you expected to happen e.g. Connector should authenticate successfully and proceed to next step to finalise the creation process. -->

## Actual Behavior
<!-- What actually happened e.g. Authentication fails and returns an 'Invalid Credentials' error. -->

## Possible Solution
<!-- Not obligatory, but suggest a fix/reason for the bug e.g. Review and update the authentication module in the BigQuery connector. -->
